cadence-style-guide.md

```diff
@@ -34,6 +34,11 @@ Comments are a vital part of smart contracts, allowing users to easily understan
 ### High level documentation at the begining of files (contracts, transactions and scripts)
 Top Level comments and comments for types, fields, events, and functions should use `///` (three slashes) because [the cadence docs generating tool](https://github.com/onflow/cadence/tree/master/tools/docgen) picks up three slash comments to auto-generate docs.
 ### Commenting functions
+Functions should be commented with a:
```

Comment:
Best to mention the documentation comments should follow the format the documentation generator supports: https://pkg.go.dev/github.com/onflow/cadence/tools/docgen#readme-documentation-comments-format.

Of course, we can also adjust/refine that format